doc/api/assert.md

```diff
@@ -26,12 +29,33 @@ When using the `strict mode`, any `assert` function will use the equality used i
 the strict function mode. So [`assert.deepEqual()`][] will, for example, work the
 same as [`assert.deepStrictEqual()`][].
 
+Besides that error messages that involve objects produce a error diff instead of
+displaying both objects. That is not the case for the legacy mode.
+
 It can be accessed using:
 
 ```js
 const assert = require('assert').strict;
 ```
 
+Example error diff:
+
+```diff
```

Comment:
1. I am not sure, but maybe it is worth to separate the input with `js` label (for better highlighting and linting) and the output with `diff` label.

2. I cannot find any `` ```diff `` insets in `.md` files in this repo or `nodejs.org` repo. Are we sure they are rendered to the `.html` docs properly?